libs/ledger-live-common/src/hw/getOnboardingStatePolling.ts

```diff
@@ -73,7 +78,6 @@ export const getOnboardingStatePolling = ({
           // Pushes the error to the next step to be processed (no retry from the beginning)
           return of(error as Error);
```

Comment:
[ASK] I guess the quit app error (if there's no app opened) is handled here?
Did you try on both LWM and LWD ? Because the custom array of allowed error is injected and it differs between the 2 apps.